.pre-commit-config.yaml

```diff
@@ -53,3 +53,11 @@ repos:
         entry: scripts/pre-commit-eslint
         language: script
         files: \.(js|jsx|ts|tsx)$
+
+  - repo: local
+    hooks:
+      - id: check-code-generation
+        name: check-code-generation
+        entry: scripts/pre-commit-check-code-generation
+        language: script
+        files: pkg/graph/schema.resolvers.go
```

Comment:
I'm making an assumption that we only need to run this code-generation check when `pkg/graph/schema.resolvers.go` is modified. Is that a correct assumption?

Reply:
For the backend, it's the ones you have:

* `pkg/graph/schema.graphql` 
* `pkg/graph/schema.resolvers.go`

For the front end, it's:

* `pkg/graph/schema.graphql` 
* `src/queries/*.ts`